slam-libraries/viam-orb-slam3/orbslam_server_v1.h

```diff
@@ -50,20 +59,34 @@ class SLAMServiceImpl final : public SLAMService::Service {
 
     // For a given GetPointCloudMapRequest
     // Returns a GetPointCloudMapResponse containing a sparse
-    // slam map as Binary PCD
-    // Map uses z axis is in the direction the camera is facing
+    // slam map as Binary PCD. The z-axis represents the direction the camera is
+    // facing at the origin of the map
     ::grpc::Status GetPointCloudMap(
         ServerContext *context, const GetPointCloudMapRequest *request,
         GetPointCloudMapResponse *response) override;
 
     // For a given GetInternalStateRequest
     // Returns a GetInternalStateResponse containing
-    // the internal state of the SLAM algorithm
-    // required to continue mapping/localization
+    // current internal state of the map represented as an ORB-SLAM Atlas(.osa)
+    // file in chunks of size maximumGRPCByteChunkSize
```

Comment:
This doesn't return chunks, but it's not so important, since this is going away soon